src/lib/y2storage/dialogs/proposal.rb

```diff
@@ -117,13 +121,22 @@ def dialog_title
         _("Suggested Partitioning")
       end
 
+      def expert_partitioner
+        items = []
+        if devicegraph
+          items << Item(Id(:expert_from_proposal), _("Start with Current Proposal"))
+        end
+        items << Item(Id(:expert_from_probed), _("Start with Existing Partitions"))
+        MenuButton(_("Expert Partitioner"), items)
```

Comment:
btw when you touch it, you can try to make openQA life easier with suggested keyboard shortcuts with "&" in name. Also I do not see it, but if final state is 2-3 options, maybe 3 push buttons instead of small menu is better? but I expect you already discuss it.
Otherwise it looks good.

Reply:
Yes, creating a menu-button was actually requested in the PBI. Looks like it's the UI suggested by the UI team.

Reply:
And whether the shortcuts should be explicitly added by us, that's relatively controversial. I understand it makes the whole thing more stable for openQA to test an English system, but I have seen some discussions about how good/bad is to enforce them at developer level instead of letting the automatic mechanisms in LibUI do their job.

Reply:
well, in past it was common practice to have UI shortcuts stable. I also personally find very annoying to have different shortcuts depending on options like it is for users ( when old system is detected, then shortcuts are different ). Automatic mechanism just ensure that there is sortcut, but there is no guarantie that it will be stable. Which we can do on other hand with giving prefered keyword there. ( as it have precedence over automatic ones, so at first it tries to assign all suggested and then find remaining ones, that is not already set ).